rules.rb

```diff
@@ -1 +1,132 @@
+  end
+end
+
+# rule 'CB003', 'CHANGELOG should follow keepachangelog.com format'
+
+rule 'CB004', 'Missing CODE_OF_CONDUCT in markdown format' do
+  tags %w(style code_of_conduct brigade)
+  cookbook do |path|
+    ensure_file_exists path, 'CODE_OF_CONDUCT.md'
+  end
+end
+
+rule 'CB005', 'Missing CONTRIBUTING in markdown format' do
+  tags %w(style contributing brigade)
+  cookbook do |path|
+    ensure_file_exists path, 'CONTRIBUTING.md'
+  end
+end
+
+### Gemfile ###
+
+# rule 'CB008', 'Gemfile should depend on berkshelf'
+# rule 'CB009', 'Gemfile should depend on chefspec'
+# rule 'CB010', 'Gemfile should depend on cookstyle'
+# rule 'CB011', 'Gemfile should depend on foodcritic'
+# rule 'CB012', 'Gemfile should depend on stove'
+# rule 'CB013', 'Gemfile should depend on test-kitchen'
+
+rule 'CB006', 'Missing LICENSE in markdown format' do
+  tags %w(style license brigade)
+  cookbook do |path|
+    ensure_file_exists path, 'LICENSE.md'
+  end
+end
+
+### README.md ###
+
+# rule 'CB007', 'README should link to chef-brigade'
+# README should link to travis
+#+[![Build Status](https://travis-ci.org/chef-brigade/mongodb-cookbook.svg)](https://travis-ci.org/chef-brigade/mongodb-cookbook)
+# README should specify maintainer
+
+### metadata.rb ###
+
+# cf FC064
+rule 'CB014', 'metadata.rb should use brigade style source_url' do
+  tags %w(metadata supermarket chef12)
+  metadata do |ast, filename|
+    name = field_value(ast, 'name')
+    expected = "https://github.com/chef-brigade/#{name}-cookbook"
+    [file_match(filename)] unless field_value(ast, 'source_url') == expected
+  end
+end
+
+# cf FC065
+rule 'CB015', 'metadata.rb should use brigade style issues_url' do
+  tags %w(metadata supermarket chef12)
+  metadata do |ast, filename|
+    name = field_value(ast, 'name')
+    expected = "https://github.com/chef-brigade/#{name}-cookbook/issues"
+    [file_match(filename)] unless field_value(ast, 'issues_url') == expected
+  end
+end
+
+rule 'CB016', 'metadata.rb should use brigade style maintainer' do
+  tags %w(metadata supermarket chef12)
+  metadata do |ast, filename|
+    expected = 'Chef Brigade'
+    [file_match(filename)] unless field_value(ast, 'maintainer') == expected
+  end
+end
+
+rule 'CB017', 'metadata.rb should use brigade style maintainer_email' do
+  tags %w(metadata supermarket chef12)
+  metadata do |ast, filename|
+    expected = 'help@chefbrigade.io'
+    [file_match(filename)] unless field_value(ast, 'maintainer_email') == expected
+  end
+end
+
+rule 'CB018', 'metadata.rb should specify compatible chef_version' do
+  tags %w(metadata supermarket chef12)
+  metadata do |ast, filename|
+    expected = '>= 11.0'
+    [file_match(filename)] unless field_value(ast, 'chef_version') == expected
+  end
+end
+
+rule 'CB019', 'metadata should not contain "attribute" keyword' do
+  tags %w(metadata supermarket chef12)
+  metadata do |ast, filename|
+    [file_match(filename)] if field(ast, 'attribute').any?
+  end
+end
```

Comment:
Most people don't put `attribute` and `recipe` directives in metadata.rb since its redundant documentation

I really only see these directives in cookbooks that are unmaintained